### DISCUSSION TITLE
LTMEAT.... Is kinda broke. Also, Is there a way to unblock Bookmartlets

### DISCUSSION THREAD

**Discussion by RavenChill-Dev** [3kh0/ext-remover#655] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
### LTMEAT
So, I've recently checked out the LTMEAT extension remover and When I tried doing it, it says connection refused when I successfully did it RIGHT. I also did another method using somewhat of the same thing, and it still says connection refused errors, and It is making me mad. Why is this happening? Could I please use the one that isn't bookmarklets? Maybe that would work. 
My other method is to go to the blocking manifest, type 
`chrome-extension://<My Blocking Extension> /manifest.json`
Type chrome://hang in the URL bar
Right click the tab and then duplicate the tab, without closing anything
Visit chrome://extensions
Click on the Extension, and then click "Allow access to files urls" 
One tab should close of the  chrome-extension://<My Blocking Extension> /manifest.json
Then everything should be unblocked, well nope. 
It worked yesterday, but now it stopped working. Know how to fix that?
### Bookmarklets
Is there a way to unblock bookmarklets?
**3kh0, I think that everything you do is amazing, and you inspire me to create things. I would Like you to please answer this.**
-Alexander Cartag

**Comment by RavenChill-Dev** [comment]
My school blocks javascript://, so is there a bypass? Also my school blocks with Lightspeed, Relay,  **AND** Go guardian. My school over does it.

**Reply by viperbot2000** [reply]
what school, same with me my school has all of them and is now using wifi blocks, starting a couple days ago

**Reply by RavenChill-Dev** [reply]
My school is Palm beach schools.

**Reply by RavenChill-Dev** [reply]
WMS

**Reply by viperbot2000** [reply]
is it by jupiter?

**Reply by RavenChill-Dev** [reply]
sort of, palm beach

**Comment by alcradite** [comment]
Well you could always use a different method like point-blank or LTBEEF inspect.

**Reply by RavenChill-Dev** [reply]
Those don't work, they also blocked my inspect element

**Reply by RavenChill-Dev** [reply]
@alcradite My school over does it.

**Comment by srets** [comment]
bro left a signature 💀

**Reply by RavenChill-Dev** [reply]
Bro, ima tryhard. Btw seeing how I;m greek, I like to do things Φαντασία

**Reply by RavenChill-Dev** [reply]
fancy*

**Reply by RavenChill-Dev** [reply]
Λοιπόν, είμαι Έλληνας, οπότε θα μπορούσατε να μου μιλήσουν ελληνικά;

**Reply by RavenChill-Dev** [reply]
I speak english fluently through.

**Comment by Hilololsusy** [comment, marked as answer]
its wifi go on your phone hotspot  have this problem too

**Reply by RavenChill-Dev** [reply]
k

**Reply by RavenChill-Dev** [reply]
i think its wifi

**Reply by RavenChill-Dev** [reply]
ty
